plugins/leemons-plugin-leebrary/frontend/src/components/AssetList.js

```diff
@@ -71,283 +52,227 @@ function getOwner(asset) {
   return !isEmpty(owner) ? `${owner?.name} ${owner?.surnames}` : '-';
 }
 
-function AssetList({
-  category: categoryProp,
-  categories: categoriesProp,
-  asset: assetProp,
-  assetType: assetTypeProp,
-  search: searchProp,
-  layout: layoutProp,
-  showPublic: showPublicProp,
-  programs,
-  subjects,
-  canChangeLayout,
-  canChangeType,
-  canSearch,
-  variant,
-  onlyThumbnails,
-  page: pageProp,
-  pageSize,
-  pageSizes,
-  published,
-  activeStatus,
+const AssetList = ({
+  categories = [],
+  category,
+  asset,
+  onSelect,
+  allowSearchByCriteria,
+  searchCriteria,
   onSearch,
-  pinned,
-  paperProps,
-  emptyComponent,
-  itemMinWidth,
-  searchEmptyComponent,
-  preferCurrent,
-  searchInProvider,
-  roles,
-  filters,
-  filterComponents,
-  allowStatusChange,
+  allowStatusFilter,
+  statusFilter,
+  onStatusChange,
   allowCategoryFilter,
-  onStatusChange = () => {},
-  onSelectItem = () => {},
-  onEditItem = () => {},
-  onTypeChange = () => {},
-  onLoading = () => {},
-}) {
+  categoryFilter,
+  onCategoryFilter,
+  allowMediaTypeFilter,
+  mediaTypes,
+  mediaTypeFilter,
+  onMediaTypeChange,
+  onEditItem,
+  allowAcademicFilter,
+  filterComponents,
+  filters: academicFilters,
+  forgetAssetToOpen,
+}) => {
+  // Setup && States
+  const session = useSession();
+  const locale = getLocale(session);
+  const [t, translations] = useTranslateLoader(prefixPN('list'));
+  const [, , , getErrorMessage] = useRequestErrorMessage();
+  const [page, setPage] = useState(1);
+  const [pageSize, setPageSize] = useState(12);
+  const [isDetailOpen, setIsDetailOpen] = useState(false);
+  const [isDrawerOpen, setIsDrawerOpen] = React.useState(false);
+  const [itemToShare, setItemToShare] = React.useState(null);
+  const [tempSearchCriteria, setTempSearchCriteria] = useState(searchCriteria);
+  const [searchCriteriaDebounced] = useDebouncedValue(tempSearchCriteria, 300);
+  const [pageAssetsData, setPageAssetsData] = useState(null);
+  const [selectedAsset, setSelectedAsset] = useState(null);
   const [cardDetailIsLoading, setCardDetailIsLoading] = React.useState(false);
-  if (categoryProp?.key?.includes('leebrary-subject')) {
-    // eslint-disable-next-line no-param-reassign
-    subjects = isArray(categoryProp.id) ? categoryProp.id : [categoryProp.id];
-  }
-
-  const location = useLocation();
 
-  const isPinsRoute = location.pathname.includes('pins');
-  const [categoryFilter, setCategoryFilter] = React.useState(null);
-
-  const [isDrawerOpen, setIsDrawerOpen] = React.useState(false);
+  const detailLabels = useMemo(() => {
+    if (!isEmpty(translations)) {
+      const items = unflatten(translations.items);
+      return items.leebrary.list.labels;
+    }
+    return {};
+  }, [JSON.stringify(translations)]);
```

Comment:
If starting from 0, it's better to use the [t]

Reply:
CardDetailWrapper necesita recibir los labels. Por esto los sacamos y se los pasamos.